### ISSUE TITLE
Change Pictures?

### ISSUE THREAD

**Issue by mccloskeymatthew91** [mfkp/tinderplusplus#39]
Hey there - I'm not sure how this app syncs up with Facebook and I don't know how to change my Tinder pictures. 

I tried deleting some old profile pictures from Facebook, but they're still on the app. Is there a code fix for that or am I doing something wrong?

Thanks!
-MattM

**Comment by beny9512** [comment]
I think, a simple cache deletion of the app should solve the problem. Maybe..

**Comment by mccloskeymatthew91** [comment]
Hey, I tried uninstalling it, deleting my browser cache and it still isn't updating my pictures - any advice?

**Comment by mrmayfield** [comment]
Matt, what is happening is Tinder is saving those pictures. This app doesn't support removing your own pictures atm, you have to do it through the mobile app.

**Comment by SaturnusDJ** [comment]
@mrmayfield is right.

Further more, cache is saved in your computer account. When having problems, remove this directory to start from scratch.
For Windows: C:\Users\*\AppData\Local\TinderPlusPlus.

**Comment by beny9512** [comment]
Hey, @mccloskeymatthew91 have you tried this option? 
![screen shot 2016-03-06 at 18 20 07](https://cloud.githubusercontent.com/assets/4943728/13555715/4b110628-e3c8-11e5-8d95-392b025dfc42.png)
cc/ @SaturnusDJ

**Comment by mccloskeymatthew91** [comment]
I'm on a PC - I've tried reinstalling the application a couple times, but
that doesn't work either. My pictures are still the same from when I first
logged in.

**Comment by mrmayfield** [comment]
And you've changed them on Tinder, such that if you log in on Tinder you can see your updates photos?

**Comment by mccloskeymatthew91** [comment]
There isn't an option to change my pictures on the Windows version of the
application. I tried changing them on Facebook, hoping they would sync with
the app, but they didn't

On Thu, Mar 24, 2016 at 10:38 PM, Anthony Mayfield <notifications@github.com

**Comment by SaturnusDJ** [comment]
@mccloskeymatthew91 
It seems some things are unclear.
The Tinder you use on the desktop (Windows, OS X, etc) is an unofficial Tinder 'program.' A lot of features that are on the Tinder mobile app (for Android and iOS) are missing in the unofficial Tinder for Windows/OS X. Changing, or adding picture is one of the missing features.

In order to change add/change/remove pictures in your Tinder profile, you **must** use Tinder on Android/iOS. As mentioned above, picture management is (currently) not possible on the unofficial Tinder for Windows/OS X.
Facebook also is not the way to go. When you select pictures from Facebook to be on your Tinder profile, they are copied from Facebook to Tinder. Removing them from Facebook has nothing to do with Tinder.

TL;DR: Changing pictures is only possible via Tinder on Android/iOS, not this desktop app, not Facebook.

**Comment by mrmayfield** [comment]
^ What Saturnus said. Open up Tinder on your mobile device either iOS or Android and change the photos there.
